MetaMask/metamask-extension
# feat: add power users survey support

## **Description**

https://github.com/user-attachments/assets/6839cf85-1a89-4c20-acdb-7dddee7349cb



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27361?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/3259

## **Manual testing steps**

1. Show toast
WHEN Popup is shown << Toast should not be shown on Notification, i.e. Confirmation screen
AND User is logged in
AND API is called with hashed user id
AND API returns PowerUser (true)
THEN Toast is shown

2. Open survey
WHEN User clicks on survey link
THEN Open url
AND Close toast

3. Close toast
WHEN User clicks close icon
THEN Close toast and don't show again

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
